docs/src/components/BuiltinSearch/index.js

```diff
@@ -5,11 +5,19 @@ import builtins from "@generated/builtin-data/default/builtins.json";
 
 import styles from "./styles.module.css";
 
-export default function BuiltinSearch({ entryLimit }) {
+export default function BuiltinSearch({ entryLimit, alwaysShow, elementId }) {
+  const hash = window.location.hash.replace("#", "").split("-");
+  console.log(hash);
   const [searchTerm, setSearchTerm] = useState("");
   const [selectedCategory, setSelectedCategory] = useState("");
+  if (hash.length == 3 && hash[0] == "builtin" && searchTerm != hash[2]) {
+    setSelectedCategory(hash[1]);
+    setSearchTerm(hash[2]);
+    window.location.hash = elementId;
+    document.getElementById(elementId)?.scrollIntoView({ behavior: "smooth", block: "start" });
+  }
```

Comment:
This is not going to be available in the pre-render step since there is no window var defined. So, we need to make this a function and wrap this component's elements in `<BrowserOnly> foo </BrowserOnly>`